samples/subsys/ipc/ipm_psoc6/README.rst

```diff
@@ -0,0 +1,74 @@
+.. _ipm-psoc6-sample:
+
+Sample mailbox application
+##########################
+
+Overview
+********
+
+The :ref:`cy8ckit_062_wifi_bt` board has two core processors (Cortex-M4
+and Cortex-M0+). This sample application uses a mailbox to send messages
+from Cortex-M4 processor core (client) to Cortex-M0+ (server).
+
+Requirements
+************
+
+- :ref:`cy8ckit_062_wifi_bt` board
+
+Building and Running
+********************
+
+#. Build the project
+
+.. code-block:: console
```

Comment:
Indent the code-block directive (and body) by three more spaces.  This will render the code-block content as part of the order list.